src/main/kotlin/org/agoranomic/assessor/decisions/cli.kt

```diff
@@ -27,6 +28,9 @@ private enum class Form(val reportConfig: ReportConfig) {
     OFFICIAL(
         ReportConfig(voteComments = false, totalBallotCount = true, voteKindBallotCount = true)
     ),
+    JSON(
```

Comment:
I would prefer this to not be a magic Form.

Reply:
Indeed, it's a little ugly. Seemed like the simplest way to fit it into your existing command-parsing architecture, but I don't like it.